web/app/scripts/filterbar/filterbar.html

```diff
@@ -11,6 +11,12 @@
                 <li ng-if="value.format=='number'" numeric-range-field label="key" data="value" class="dropdown"></li>
                 <li ng-if="value.fieldType=='selectlist'" options-field label="key" data="value" class="dropdown"></li>
             </ul>
+            <ul class="nav navbar-nav navbar-right">
+                <!-- TODO: this is a temporary location for the Add Record button which has been removed -->
```

Comment:
This TODO can safely be removed. It's in a good location now, and there's an issue open for styling it better.